dashboard/app/models/census/census_summary.rb

```diff
@@ -146,13 +147,138 @@ def self.decide_teaches(posterior)
   end
 
   def self.summarize_school_data(school, school_years, years_with_ap_data, years_with_ib_data, state_years_with_data)
+    summarize_school_data_simple(school, school_years, years_with_ap_data, years_with_ib_data, state_years_with_data)
+  end
+
+  def self.summarize_school_data_simple(school, school_years, years_with_ap_data, years_with_ib_data, state_years_with_data)
+    summaries = []
+
+    school_years.each do |school_year|
+      audit = {
+        version: 0.3,
+        stats: {},
+        census_submissions: [],
+        ap_cs_offerings: [],
+        ib_cs_offerings: [],
+        state_cs_offerings: [],
+      }
+
+      count_yes = 0
+      count_no = 0
+
+      # If the school doesn't have stats then treat it as not high school.
+      # The lack of stats will show up in the audit data as a null value for high_school.
```

Comment:
Same for K8 school, given the lines below that do the same thing as high school?

Reply:
Updated the comment to mention k8_schools too.